index.js

```diff
@@ -9,13 +9,27 @@ exports.prepare = function (extensions, filepath, cwd, nothrow) {
   var attempts = [];
   var err;
   var onlyErrors = false;
-  var ext = extension(filepath);
-  if (Object.keys(require.extensions).indexOf(ext) !== -1) {
+  var exts = extension(filepath);
+
+  var supportsExtension = Object.keys(require.extensions).indexOf(exts[0]) !== -1;
```

Comment:
In the context of Liftoff, it was wrong of rechoir to throw on an unknown extension. An unknown extension could very well be standard JS (as is the case for @alansouzati). I've fixed this here: https://github.com/tkellen/js-rechoir/commit/62fb2a62b6102f9ba07a05c7380ce237bb47b349

We still need to use some portion of this PR to support things like `tmp.babel.js`, though.